src/RunspaceProviders/Kubernetes/VMware.ScriptRuntimeService.K8sRunspaceProvider/K8sRunspaceProvider.cs

```diff
@@ -498,95 +508,31 @@ public class K8sRunspaceProvider : IRunspaceProvider {
 
       public IRunspaceInfo WaitCreateCompletion(IRunspaceInfo runspaceInfo) {
          var result = runspaceInfo;
```

Comment:
Remove the result variable and use return statements. The code, as it is, has too many code branches to follow.

Reply:
fixed